.github/workflows/ci.yml

```diff
@@ -73,6 +80,24 @@ jobs:
         command: build
         args: --verbose --target thumbv6m-none-eabi --no-default-features --features random
 
+    - name: Build WASM
```

Comment:
Isn't this redundant to `thumbv6m-none-eabi`? `thumbv6m-none-eabi` is even more strict than wasm because it won't allow any pull from stdlib whereas `wasm32-unknown-unknown` is more permissive.

This isn't a blocker ofc and doesn't hurt to include an additional check, but its worthy to mention this

Reply:
It is in fact redundant when I look at it this way. Still, I'd say we keep a `wasm32-unknown-unknown` build for posterity.